.openshift-ci/ci_tests.py

```
@@ -235,6 +235,20 @@ def run(self):
         )
 
 
+class VMScanningE2e(BaseTest):
+    TEST_TIMEOUT = 2 * 60 * 60
+    TEST_OUTPUT_DIR = "/tmp/vm-scanning-test-logs"
```

Ruff (0.15.10)

[error] Probable insecure usage of temporary file or directory: "/tmp/vm-scanning-test-logs"

(S108)